docs/release-notes/1.10.0.md

```diff
@@ -14,6 +14,7 @@
 * {func}`scanpy.pp.pca`, {func}`scanpy.pp.scale`, {func}`scanpy.pl.embedding`, and {func}`scanpy.experimental.pp.normalize_pearson_residuals_pca`
   now support a `mask` parameter {pr}`2272` {smaller}`C Bright, T Marcella, & P Angerer`
 * {func}`scanpy.tl.rank_genes_groups` no longer warns that it's default was changed from t-test_overestim_var to t-test {pr}`2798` {smaller}`L Heumos`
+* {func}`scanpy.pp.highly_variable_genes` supports dask for the default `seurat` flavor and partially for the `cell_ranger` flavor {pr}`2809` {smaller}`P Angerer`
```

Comment:
Is it still true that it's only partially for the `"cell_ranger"` flavor?

Reply:
good call, full support!